privacy.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: page
+title: Data Privacy
+tagline: Data Privacy information for Jupyter Related software
+permalink: /privacy
+---
+
+
+# F.A.Q
+
+This page tried to answer regularly asked question about Jupyter and Data Privacy.
+
+## Does Project Jupyter **Software** Collect information ? 
+
+Project Jupyter Software does not collect any information about its installation or usage. 
+
+All the software that is developed by the Jupyter Project are free of tracking
+code. When you install and deploy Jupyter add related libraries on your own
+computer or servers no informations is collected not exchanged with the Jupyter
+Developer.
+
+We can only guaranty this is true when you use Jupyter Directly and not via a
+Third Party Vendor that provide hosted Jupyter Notebook, if you have questions
+about hosted Jupyter please see your vendor documentation and privacy sections.
+
+You can also look at the [JupyterLab specific privacy
+page](https://jupyterlab.readthedocs.io/en/stable/privacy_policies.html).
+
+## Does Jupyter Project **services** Collect information ?
+
+Project Jupyter run a few free services related to Jupyter, this is include but
+not limited to nbviewer, Binder, and JupyterLite. 
+
+Those services are not designed to stored any user data on Jupyter ownd
+infrastructure. Though being publicly accessible we caution user to not use
+those services to process sensitive informations.
```

Comment:
```suggestion
These services do not store any user data on Jupyter-owned
systems. Because these services are publicly accessible, we caution users not to use
those services to process sensitive information.
```

Reply:
I was trying to be a bit careful with this one as I don't know how those are ran, and I'm guessing there might be logs from nginx that keeps IP or alike that may be PII in some context, though I guess this is not "user data".